app/services/cull_stale_assessments_service.rb

```diff
@@ -0,0 +1,10 @@
+class CullStaleAssessmentsService
+  def self.call
+    new.call
+  end
+
+  def call
+    assessments = Assessment.where(created_at: 10.years.ago..2.weeks.ago)
+    assessments.map(&:destroy!)
```

Comment:
would `destroy_all` not work here?

```suggestion
    assessments.destroy_all
```

Or even

```suggestion
   Assessment.destroy_by(created_at: 10.years.ago..2.weeks.ago)
```

Reply:
You can also use infinite ranges i believe.
```suggestion
  Assessment.destroy_by(created_at: ..2.weeks.ago)
```